FrontEnd/index.html

```diff
@@ -15,7 +15,12 @@
                         </a>
                     </div>
                     <div class="float-right">
```

Comment:
the idea behind is nice, it will be nicer if we can find better name for "float-right"